### DISCUSSION TITLE
Flathub redeclipse install Problem

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1263] [Help and Support] [answered]
flatpak install flathub net.redeclipse.RedEclipse
Looking for matches…

net.redeclipse.RedEclipse permissions:
    ipc   network   pulseaudio   x11   devices

        ID                                  Branch          Op          Remote           Download
 1. [✗] net.redeclipse.RedEclipse           stable          i           flathub          1,0 kB / 937,3 MB

Error: Failed to read commit beb6f924e3baed7301fc9018da22f7bf98266ca4c40a808d16031d68eed2f60b: No such metadata object ad673760a8c24467e3fe2c7efe835371a25dcc5d81e3956e6757e00a4a5787d8.dirtree
error: Failed to install net.redeclipse.RedEclipse: Failed to read commit beb6f924e3baed7301fc9018da22f7bf98266ca4c40a808d16031d68eed2f60b: No such metadata object ad673760a8c24467e3fe2c7efe835371a25dcc5d81e3956e6757e00a4a5787d8.dirtree

Thanks

**Comment by qreeves** [comment, marked as answer]
Red Eclipse does not have an official flathub image, please see the website you obtained it from to contact the maintainer of that package.
